core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -77,11 +77,33 @@
 
   /** Conformance value that instructs Calcite to use SQL semantics
    * consistent with Microsoft SQL Server version 2008. */
-  SQL_SERVER_2008;
+  SQL_SERVER_2008,
 
+  /** Conformance value that instructs Calcite to use SQL semantics
+   * consistent with Dialect1. */
+  DIALECT_1,
+
+  /** Conformance value that instructs Calcite to use SQL semantics
+   * consistent with Hive. */
+  HIVE,
+
+  /** Conformance value that instructs Calcite to use SQL semantics
+   * consistent with PostgreSQL. */
+  POSTGRESQL,
+
+  /** Conformance value that instructs Calcite to use SQL semantics
+   * consistent with Redshift. */
+  REDSHIFT,
+
+  /** Conformance value that instructs Calcite to use SQL semantics
+   * consistent with MySQL version 8.x. */
+  MYSQL_8;
+
+  // TODO: change to isLeftSemiJoinAllowed, only place it's used. Remove BABEL.
```

Comment:
nit: Are we going to deal with this TODO?

Reply:
I forgot about that, did we want to remove the BABEL SqlConformanceEnum?

Reply:
Yes, let's remove it.

Reply:
In fact, we should double check that there are no references to "server" or "babel" throughout the entire codebase.

Reply:
In places like [this](https://github.com/googleinterns/calcite/blob/master/core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java#L481), should we use like the LENIENT conformance enum?

Reply:
Yeah, let's use LENIENT.